mobile/src/main/java/org/openhab/habdroid/ui/AboutActivity.kt

```diff
@@ -113,6 +114,14 @@ class AboutActivity : AbstractBaseActivity(), FragmentManager.OnBackStackChanged
                 .desc(context.getString(R.string.about_copyright, year))
                 .icon(R.mipmap.icon)
                 .build())
+            if (Util.isFlavorStable) {
```

Comment:
I wonder whether we only should do this in the full-stable flavor, since people probably use the foss flavor because they don't want to use Google's services.

Reply:
As above